docs/docs/components/button.md

```diff
@@ -34,6 +34,9 @@ delayLongPress: number = 1000;
 // If disabled, touch and mouse input events are ignored
 disabled: boolean = false;
 
+// By default, opacity of a disabled element is 0.5. This value can be overriden with this property
+disabledOpacity: number = undefined;
```

Comment:
If you're adding a new prop, it needs to be documented in Button.md and added to the RXPTest app.

Reply:
Ive already added it to button.md -> will add it to the test

Reply:
@erictraut i have added this change to the test - any thing else is required from my side?